packages/datadog-instrumentations/src/child_process.js

```diff
@@ -84,18 +130,59 @@ function wrapChildProcessCustomPromisifyMethod (customPromisifyMethod, shell) {
 
     const childProcessInfo = normalizeArgs(arguments, shell)
 
-    return childProcessChannel.tracePromise(
-      customPromisifyMethod,
-      {
-        command: childProcessInfo.command,
-        shell: childProcessInfo.shell
-      },
-      this,
-      ...arguments)
+    const context = {
```

Comment:
`tracePromise` does not support `abortController` to prevent call to the original function and reject a promise